fw/http_limits.c

```diff
@@ -194,6 +194,9 @@ frang_conn_new(struct sock *sk)
 	/* The new socket is allocated by inet_csk_clone_lock(). */
 	assert_spin_locked(&sk->sk_lock.slock);
 
+	if (tfw_http_mark_is_in_whitlist(skb->mark))
+		return T_OK;
```

Comment:
We don't need to lookup for a vhost above in this code path and I'm not sure if the compiler will optimize the call out, so it's better to move `dflt_vh` to the below of the check

Reply:
Yes fixed